jobdsl/organisations-beta.groovy

```diff
@@ -24,7 +24,7 @@ List<Map> orgs = [
         [name: 'IAC', regex: 'ia.*'],
         [name: 'IDAM', regex: '(idam-.*|cnp-idam-.*)'],
         [name: 'MI', displayName: 'Management Information'],
-        [name: 'PCQ', regex: 'pcq-.*'],
+        [name: 'HMI', regex: 'hmi-.*'],
```

Comment:
you don't need a regex if it matches your name, and you probably want a display name if your team name isn't evident from the short name

Reply:
Removed the regex.